### ISSUE TITLE
Add zipped and unzipped sizes to the GTFS dataset

### ISSUE THREAD

**Issue by davidgamez** [MobilityData/mobility-feed-api#650]
### Describe the problem

We should add the size of the zip and unzipped versions to the dataset's metadata. For producers, this can help them make storage requirement analyses without having to download the file. For _internal_  use, we can make better decisions regarding storage needs and tune the data pipeline to use bigger VMs for bigger feeds.

### Proposed solution

Add zipped and unzipped sizes to the GTFS dataset table.

**Comment by emmambd** [comment]
@davidgamez is this worth including as part of the "unfetchable" milestone?

**Comment by davidgamez** [comment]
> @davidgamez is this worth including as part of the "unfetchable" milestone?

Yes, it can be part of the unfetchable milestone. On a different topic, I haven't thought about how to have this information rendered in the UI.

**Comment by emmambd** [comment]
TransitFeeds had it associated with each individual dataset, which I think makes the most sense: https://transitfeeds.com/p/128-business-council/763

**Comment by davidgamez** [comment]
Tasks:
- Capture zipped size and add it as part of the dataset information.
- Serve the size information in the API(dataset entity).
- Create a script to capture the already downloaded datasets.
